python/pminit/pminit/cli.py

```diff
@@ -0,0 +1,43 @@
+import os, sys
+import subprocess
+import argparse
+
+def execute(cmd: str, cwd: str):
+    popen = subprocess.Popen(cmd, stdout = subprocess.PIPE, stderr = subprocess.STDOUT,
+        shell = True, text = True, cwd = cwd )
+    for stdout_line in iter(popen.stdout.readline, ""):
+        sys.stdout.write(stdout_line)
+        sys.stdout.flush()
+
+    popen.stdout.close()
+    return_code = popen.wait()
+    #For some reason npm returns a non zero
+    #exit code when `--help` is used in the commands.
+    #This shouldn't raise an error.
```

Comment:
`npm --help` returns non-zero exit code, but `npm install --help` exists with zero exist code.

Could we confirm that is a intentional behaviour with the npm team?

Reply:
`--help` is actually not a recognized flag for bare `npm`. It's `npm help`

`npm --help` has the same behaviour as `npm --random-flag`
```bash
npm --help; echo $? # exit code 1
npm --random-flag; echo $? # exit code 1
```